packages/jest-reporters/src/types.ts

```diff
@@ -63,3 +63,7 @@ export type SummaryOptions = {
   showSeed?: boolean;
   seed?: number;
 };
+
+export type SummaryReporterOptions = {
```

Comment:
Perhaps this type should be moved to `SummaryReporter.ts` file? It is not shared between the modules, so it makes sense to define it near the `SummaryReporter` class.